md/getting-started-tutorial.md

```diff
@@ -39,10 +38,10 @@ Remember to click **_Save_** to save your work frequently.
 
 In Bonita BPM Studio, start the UI Designer by clicking the icon in the coolbar. This opens the UI Designer in your browser, at the home page.
 
-In the UI Designer home page, click on the **_Create_** button. This brings a pop-up.  
-In the pop-up, set the type to _Application page_ and name to _TravelTool_.  
-The name must not contain spaces or special characters.  
-This name is used in Bonita BPM to identify the page. It is not displayed to the application user.   
```

Comment:
Should be ok now. Still it seems that line break are ignore for content within a list (not a new issue).